qgisfeedproject/qgisfeed/views.py

```diff
@@ -177,4 +177,63 @@ def feeds_list(request):
               "form": form,
               "count": count
             }
-    )
\ No newline at end of file
+    )
+
+
+@staff_required
+def feed_entry_add(request):
```

Comment:
@Xpirix can you update this to use django class based views? I will merge this for now, you can create a new PR

Reply:
@dimasciput Okay, thanks. I created a new ticket for it.